### PR TITLE
fix: cannot match * on windows

### PR DESCRIPTION
**Pull request by FrankCheungCN**
**situations:** 
When I use `jszip-cli add example/* -o ./example.zip` on windows, it just pack an empty file.

**changes:**
use glob to support matching * on windows

### CODE REVIEW

File: packages/jszip-cli/src/BuildService.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import * as path from 'path';
 import * as progress from 'progress';
 import {FileService} from './FileService';
 import {Entry, TerminalOptions} from './interfaces';
+import * as glob from 'glob';
```
Comment (ffflorian) [inline packages/jszip-cli/src/BuildService.ts:8]:
```suggestion
import {globSync} from 'glob';
```

File: packages/jszip-cli/src/BuildService.ts [hunk 2]
```diff
@@ -42,7 +43,8 @@ export class BuildService {
 
   public add(rawEntries: string[]): BuildService {
     this.logger.info(`Adding ${rawEntries.length} entr${rawEntries.length === 1 ? 'y' : 'ies'} to ZIP file.`);
-    this.entries = rawEntries.map(rawEntry => {
+    const normalizedEntries = this.normalizePaths(rawEntries);
+    this.entries = glob.globSync(normalizedEntries).map(rawEntry => {
```
Comment (ffflorian) [inline packages/jszip-cli/src/BuildService.ts:47]:
```suggestion
    this.entries = globSync(normalizedEntries).map(rawEntry => {
```

File: packages/jszip-cli/src/BuildService.ts [hunk 3]
```diff
@@ -73,6 +75,15 @@ export class BuildService {
     return this;
   }
 
+  /**
+   * Note Glob patterns should always use / as a path separator, even on Windows systems,
```
Comment (ffflorian) [inline packages/jszip-cli/src/BuildService.ts:79]:
```suggestion
   * Note: glob patterns should always use / as a path separator, even on Windows systems,
```

**Review by ffflorian — CHANGES_REQUESTED** [review]

**Comment by ffflorian** [comment]
Thanks for the PR @FrankCheungCN! I added a few comments, but beside that it looks good already. I also like that glob version 10.x works without setting `esModuleInterop` to `true` in `tsconfig.json`.

File: packages/jszip-cli/src/BuildService.ts (hunk 3 above)
Comment (FrankCheungCN) [reply packages/jszip-cli/src/BuildService.ts]: That's cool! I had commited the changes. And I think glob@10 works fine without setting `esModuleInterop`.

**Comment by FrankCheungCN** [comment]
That's cool! I had commited the changes. And I think glob@10 works fine without setting `esModuleInterop`.

**Comment by ffflorian** [comment]
Thank you!

**Pull request merged**